README.md

```diff
@@ -18,26 +17,64 @@ If you have any trouble, we will have facilitators on-site as early as 8:30 AM l
 
 As an alternative, a workshop session can be run on mybinder.org via this link: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/astropy/astropy-workshop/HEAD)
```

Comment:
Does the Binder thingy apply for this workshop? If not, we should remove.

Reply:
binder should be a very last-ditch thing because it takes much longer to get going than it used to. We would also need to update it for this branch, so I'll take it out for now